<launch>

    <!-- Arguments-->
    <arg name="start_x" default="0.03374290466308594"/>
    <arg name="start_y" default="-0.0013123006792739"/>
    <!-- <arg name="start_x" default="2.44"/>
    <arg name="start_y" default="-0.75"/> -->
    <arg name="start_z" default="0.0"/>
    <!-- <arg name="goal_x" default="3.6837387084960938"/>
    <arg name="goal_y" default="-3.395461082458496"/> -->
    
    <!-- <arg name="goal_x" default="1.62770414352417"/>
    <arg name="goal_y" default="-3.690054416656494"/> -->

    <arg name="goal_x" default="2.4167144298553467"/>
    <arg name="goal_y" default="-5.35239315032959"/>


    <include file="$(find maze_pkg)/launch/maze.launch">
        <arg name="maze_id" value="10"/>
        <param name="x_pos" value="$(arg start_x)"/>
        <param name="y_pos" value="$(arg start_y)"/>
        <param name="z_pos" value="$(arg start_z)"/>
        <param name="goal_x" value="$(arg goal_x)"/>
        <param name="goal_y" value="$(arg goal_y)"/>
    </include>
    
    <!-- Unknown Maze Navigator Node -->
    <node pkg="maze_bot_control" type="aufgabe_2.py" name="aufgabe2" output="screen">
        <param name="start_x" value="$(arg start_x)"/>
        <param name="start_y" value="$(arg start_y)"/>
        <param name="start_z" value="$(arg start_z)"/>
        <param name="goal_x" value="$(arg goal_x)"/>
        <param name="goal_y" value="$(arg goal_y)"/>
    </node>
</launch>